src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -285,7 +285,7 @@
 		}
 
 		private setNameInternal(elementId: number, name: string): void {
-			this.getView(elementId).setAttribute("XamlName", name);
+			this.getView(elementId).setAttribute("xamlname", name);
```

Comment:
?

Reply:
Selenium's lookup is case sensitive, this is an alignment with another part where this property is already lower cased.